custom_components/bodymiscale/sensor.py

```diff
@@ -81,7 +81,7 @@ async def async_setup_entry(
                 translation_key="weight",
                 icon="mdi:weight-kilogram",
                 native_unit_of_measurement="kg",
-                SensorDeviceClass.WEIGHT,
+                device_class="weight",
```

Comment:
`device_class=SensorDeviceClass.WEIGHT,` 
It's good after add `SensorDeviceClass,` to `from homeassistant.components.sensor import`

```
from homeassistant.components.sensor import (
    SensorDeviceClass,
    SensorEntity,
    SensorEntityDescription,
    SensorStateClass,
)
```

I'm going to update with it soon but I need to continue testing with the `lb` unit.

Reply:
I sent the sensor file. If you want to test it.